apps/distgo/pkg/script/script.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 func Write(buildSpec params.ProductBuildSpec, script string) (name string, cleanup func() error, rErr error) {
-	tmpFile, err := ioutil.TempFile(buildSpec.ProjectDir, "")
+	tmpFile, err := ioutil.TempFile(buildSpec.Build.OutputDir, "distgo-script-")
```

Comment:
Is this directory guaranteed to exist?

Reply:
According to the test failures, apparently not. Is there a reason we don't want to use the host's TMDIR by leaving this argument empty?